### PR TITLE
CR#1087576: add soft kernel instance to name in query cmd

### PR DESCRIPTION
**Pull request by sarab96**
add soft kernel instance to name in query cmd
New output:
...
SCU[26]: kernel_vcu_decoder_26           @N/A               (IDLE)        N/A
SCU[27]: kernel_vcu_decoder_27           @N/A               (IDLE)        N/A
SCU[28]: kernel_vcu_decoder_28           @N/A               (IDLE)        N/A
SCU[29]: kernel_vcu_decoder_29           @N/A               (IDLE)        N/A
SCU[30]: kernel_vcu_decoder_30           @N/A               (IDLE)        N/A
SCU[31]: kernel_vcu_decoder_31           @N/A               (IDLE)        N/A
SCU[32]: kernel_vcu_encoder_0            @N/A               (IDLE)        N/A
SCU[33]: kernel_vcu_encoder_1            @N/A               (IDLE)        N/A
SCU[34]: kernel_vcu_encoder_2            @N/A               (IDLE)        N/A
SCU[35]: kernel_vcu_encoder_3            @N/A               (IDLE)        N/A
SCU[36]: kernel_vcu_encoder_4            @N/A               (IDLE)        N/A
...

### CODE REVIEW

File: src/runtime_src/core/pcie/tools/xbutil/xbutil.h [hunk 1]
```diff
@@ -501,7 +501,8 @@ class device {
             uint32_t status = parseComputeUnitStat(custat, i, cu_stat::stat);
             uint32_t usage = parseComputeUnitStat(custat, i, cu_stat::usage);
             std::string name = psKernels.at(psk_inst).pkd_sym_name;
-            name += ":scu_" + std::to_string(i - computeUnits.size());
+            //name += ":scu_" + std::to_string(i - computeUnits.size());
```
Comment (larry9523) [inline src/runtime_src/core/pcie/tools/xbutil/xbutil.h:504]: Should we remove this line instead of comment it out?

**Review by larry9523 — COMMENTED** [review]
Looks good. Only nit is we probably better remove line 504. It is not needed any more.

File: src/runtime_src/core/pcie/tools/xbutil/xbutil.h (hunk 1 above)
Comment (sarab96) [reply src/runtime_src/core/pcie/tools/xbutil/xbutil.h]: done

**Comment by sarab96** [comment]
Done
> Looks good. Only nit is we probably better remove line 504. It is not needed any more.

**Review by larry9523 — APPROVED** [review]

**Pull request merged**